pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/ColumnMetadata.java

```diff
@@ -88,6 +94,29 @@ public static ColumnMetadata fromPropertiesConfiguration(String column, Properti
     }
     builder.setPaddingCharacter(paddingCharacter);
 
+    // derived metric
```

Comment:
comment will be more useful if you indicate what metadata property it's reading to check for derived type